composer.json

```
@@ -12,7 +12,7 @@
         "cweagans/composer-patches": "^1 || ^2",
         "php-http/logger-plugin": "^1",
         "php-http/message": "~1.13",
-        "phpro/soap-client": "~2.4",
+        "phpro/soap-client": ">=3 <4",
```

Hi,

what is the difference between `>=3 <4` and `^3`

```suggestion
        "phpro/soap-client": ">=3 <4",
```

---

Hello,

No difference, it's just that `^3` is the shorter writing of `>=3 <4`. 
Do you prefer the shorter writing ?

---

Hi,

Thanks for the clarification! I don’t have a strong preference personally, but it does seem like the ^3 syntax is more widely adopted in the PHP community. Given the Minimum Description Length principle, which suggests using the most concise and clear format, it might make sense to stick with the shorter ^3 notation.

Thanks again for the explanation!

---

I changed it to the shorter writing.

---

Thanks!